features/step_definitions/element_steps.rb

```diff
@@ -76,7 +86,9 @@
   expect(@test_site.home.retiring_element).not_to be_visible
 end
 
-Then('I do not wait for an nonexistent element when checking for invisibility') do
+Then(
+  'I do not wait for an nonexistent element when checking for invisibility'
```

Comment:
Lets alter this cuke name to something not as silly sounding.

Something like `'I stop waiting once the element is invisible'`

Reply:
Can we tackle this here?